task3/frontpage.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="frontpage.css"> 
+<head>
+   <!-- <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">-->
+    <title>NTW</title>
+</head>
+<body  >
+    <header class="blok "  >
+        
+        <div class="d1 box0"></div>
```

Comment:
The purpose of that block is unclear...
If that just space in using layout abilities - ok but in the real world tru to avoid such things.